cpp/include/Ice/IncomingRequest.h

```diff
@@ -30,8 +30,8 @@ namespace Ice
          * @param adapter The object adapter to set in Current.
          * @param inputStream The input stream buffer over the incoming Ice protocol request message. The stream is
          * positioned at the beginning of the request header - the next data to read is the identity of the target.
-         * @param batchRequestCount When the incoming request is a batch request with two or more requests, provides the
-         * total number of requests in the batch request.
+         * @param requestCount The total number of requests contained in the InputStream. The value is always 1 for
```

Comment:
This does not match the implementation.

In the implementation, requestCount is the number of remaining requests in inputStream. And 1 means either non-batch or the last request of a batch.